openshift/pgweb-deployment.yaml

```diff
@@ -0,0 +1,150 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: postgres
+  annotations:
+    description: "Thoth: PGWeb"
+    openshift.io/display-name: "Thoth: PGWeb"
+    version: 0.4.2
+    tags: thoth,ai-stacks,postgresql,pgweb
+    template.openshift.io/documentation-url: https://github.com/Thoth-Station/
+    template.openshift.io/long-description: >
+      This template defines PGWeb needed to browse PostgreSQL content in Thoth on OpenShift.
+    template.openshift.io/provider-display-name: "Red Hat, Inc."
+  labels:
+    app: thoth-postgres
+    component: postgres
+
+objects:
+
+- kind: Route
+  apiVersion: v1
+  metadata:
+    name: pgweb
+    labels:
+      app: thoth-postgres
+      component: pgweb
+  spec:
+    to:
+      kind: Service
+      name: pgweb
+
+- kind: Service
+  apiVersion: v1
+  metadata:
+    labels:
+      service: pgweb
+      app: thoth-postgres
+      component: pgweb
+    name: pgweb
+  spec:
+    ports:
+    - port: 80
+      protocol: TCP
+      targetPort: 8081
+    selector:
+      service: pgweb
+
+- apiVersion: v1
+  kind: DeploymentConfig
+  metadata:
+    labels:
+      service: pgweb
+      app: thoth-postgres
+      component: pgweb
+    name: pgweb
+  spec:
+    replicas: 1
+    selector:
+      service: pgweb
+    template:
+      metadata:
+        labels:
+          service: pgweb
+      spec:
+        containers:
+        - env:
+          - name: DB_HOST
+            value: postgresql
+          - name: DB_PORT
+            value: "5432"
+          - name: DB_DATABASE
```

Comment:
isnt there a configmap we can read them from?!

Reply:
Yes, this was a quick and dirty quick thing. I'll create an ansible role to provision PG from service catalog and adjust this accordingly (add secrets and cm).